### PR TITLE
moveing format specific build descriptions content

### PR DESCRIPTION
**Pull request by adrianschroeter**
It is not special to osc local builds, but are generic funcitonality,
including on server side.

### CODE REVIEW

File: xml/obs_supported_formats.xml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     OBS specific RPM macros which are set are: <literal>%_project</literal> and <literal>%_repository</literal>
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     #!BuildConflicts:
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:36]:
```suggestion
     <code>#!BuildConflicts:</code>
```

File: xml/obs_supported_formats.xml [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     #!BuildConflicts: 
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     #!BuildIgnore:
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:41]:
```suggestion
     <code>#!BuildIgnore: </code>
```

File: xml/obs_supported_formats.xml [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     #!BuildIgnore: 
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     #!BuildRequires:
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:46]:
```suggestion
     <code>#!BuildRequires:</code>
```

File: xml/obs_supported_formats.xml [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+   -->
+   <para>
+	&kiwi; appliance builds create images which can be used for direct consument. Note: builds in non-VM 
+	environments do fail often due to the bootloader setup. Use osc build --vm-type=kvm for local building.
+	OBS is evaluating kiwi builds at least two times. One time for the build host dependencies and another
+	time for the target distribution used inside of the image. For container builds a third evaluation
+	happens to find the base container.
+   </para>
+   <para>
+     kiwi builds inside of OBS need to fullfil following requirements:
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:108]:
```suggestion
     &kiwi; builds inside of OBS need to fullfil following requirements:
```

File: xml/obs_supported_formats.xml [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+    * Repos needs to be specified as obs://$PROJECT/$REPO or using http:// Urls which can be mapped
+      by OBS.
+    * Content of root directory as root.tar.bz2 or root.obscpio
+    * If you get the error "have choice", select a package and add it to
+      the package list
+    * 
+   -->
+   <para>
+	&kiwi; appliance builds create images which can be used for direct consument. Note: builds in non-VM 
+	environments do fail often due to the bootloader setup. Use osc build --vm-type=kvm for local building.
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:102]:
```suggestion
	environments do fail often due to the bootloader setup. Use <command>osc build --vm-type=kvm</command> for local building.
```

File: xml/obs_supported_formats.xml [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+	OBS is evaluating kiwi builds at least two times. One time for the build host dependencies and another
+	time for the target distribution used inside of the image. For container builds a third evaluation
+	happens to find the base container.
+   </para>
+   <para>
+     kiwi builds inside of OBS need to fullfil following requirements:
+   </para>
+  <itemizedlist>
+   <listitem>
+     <para>config.xml files need to be renamed to a filename with .kiwi suffix. </para>
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:112]:
```suggestion
     <para><filename>config.xml</filename> files need to be renamed to a filename with <filename class="extension">.kiwi</filename> suffix. </para>
```

File: xml/obs_supported_formats.xml [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+   </listitem>
+   <listitem>
+     <para>repositories which are used must use either a URL which starts with the OBS download prefix or they must
+  	   be written in obs://PROJECT/REPOSITORY syntax. </para>
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:116]:
```suggestion
  	   be written in <uri>obs://PROJECT/REPOSITORY</uri> syntax. </para>
```

File: xml/obs_supported_formats.xml [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+  </itemizedlist>
+
+   <para>
+     OBS extends kiwi functionality with following options. These covers cases which would need explicit 
+     command line commands, so they are covered via the tags to have them in a reproducible way.
+   </para>
+  
+  <itemizedlist>
+   <listitem>
+     <para>A repository defined as obsrepositories:/ will used the expanded list of repositories as defined in the proejct meta.
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:127]:
```suggestion
     <para>A repository defined as <uri>obsrepositories:/</uri> will used the expanded list of repositories as defined in the project meta.
```

File: xml/obs_supported_formats.xml [hunk 9]
```diff
@@ -0,0 +1,246 @@
...
+   </listitem>
+   <listitem>
+     <para>Container builds can specify a specific repository which shall be used for searching the base container. 
+  	   Use the xml comment <command>OBS-Imagerepo:$REPOSITORY_URL</command> for this. The repository url may be
+  	   in obs://$PROJECT/$REPOSITORY style.
+     </para>
+   </listitem>
+   <listitem>
+     <para>Container builds can specify additional tags via <command>OBS-AddTag:$TAG</command>. (kiwi limits it to one tag)
+  	   in obs://$PROJECT/$REPOSITORY style.
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:158]:
```suggestion
  	   in <uri>obs://$PROJECT/$REPOSITORY</uri> style.
```

File: xml/obs_supported_formats.xml [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+   <listitem>
+     <para>Packages get picked usually in a fixed order from the most important repository in &obsa;. This happens independend 
+  	 of the version number of the packages. Set the xml comment <command>OBS-UnorderedRepos</command> to disable the repository order handling and to have a more similar behaviour as with plain package manager builds. Note: this can result into switching 
+  	 repositories between builds and is therefore less reproducible.
+     </para>
+   </listitem>
+   <listitem>
+     <para>Container builds can specify a specific repository which shall be used for searching the base container. 
+  	   Use the xml comment <command>OBS-Imagerepo:$REPOSITORY_URL</command> for this. The repository url may be
+  	   in obs://$PROJECT/$REPOSITORY style.
```
Comment (tomschr) [inline xml/obs_supported_formats.xml:153]:
```suggestion
  	   in <uri>obs://$PROJECT/$REPOSITORY</uri> style.
```

**Review by tomschr — CHANGES_REQUESTED** [review]
@adrianschroeter Just a very rough review. Most are just formal things. I haven't done a through text review. Take what you think is useful and merge it afterwards. :-) :+1: We can improve the text in a separate PR.

**Comment by adrianschroeter** [comment]
thanks, done!

**Pull request merged**